modules/ROOT/pages/sftp/sftp-documentation.adoc

```diff
@@ -106,189 +110,233 @@ An FileSystemProvider which provides instances of SftpFileSystem from instances
+Copies the file at the `sourcePath` into the `targetPath`. 
+
+If `targetPath` doesn't exist, and neither does its parent, then an attempt is made to create a parent folder depending on the value of the `createParentFolder` argument. If `createParentFolder` is set to `false`, then an `SFTP:ILLEGAL_PATH` error is thrown. 
+
+If the overwrite argument is set to `true` and the target file already exists, then the target file is overwritten. Otherwise, an `SFTP:FILE_ALREADY_EXISTS` error is thrown. 
+
+The `sourcePath` can be either a file or a directory. If it points to a directory, then it is copied recursively.
 
-If the target file already exists, then it is overwritten if the overwrite argument is true. Otherwise, a SFTP:FILE_ALREADY_EXISTS error is thrown. As for the sourcePath, it can either be a file or a directory. If it points to a directory, then it is copied recursively.
 
 ==== Parameters
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Source Path a| String |  +++The path to the file to be copied.+++ |  | *x*{nbsp}
-| Target Path a| String |  +++The target directory where the file is going to be copied.+++ |  | *x*{nbsp}
-| Create Parent Directories a| Boolean |  +++Whether or not to attempt creating any parent directories which don't exist.+++ |  +++true+++ | {nbsp}
-| Overwrite a| Boolean |  +++Whether or not to overwrite the file if the target destination already exists.+++ |  +++false+++ | {nbsp}
-| Rename To a| String |  +++Copied file's new name. If not provided, original file name is kept.+++ |  | {nbsp}
+| Source Path a| String |  The path to the file to copy. |  | *x*{nbsp}
+| Target Path a| String |  The target directory for where to copy the file. |  | *x*{nbsp}
+| Create Parent Directories a| Boolean |  Whether or not to attempt creating any parent directories that don't exist. |  true | {nbsp}
+| Overwrite a| Boolean |  Whether or not to overwrite the file if the target destination already exists. |  false | {nbsp}
+| Rename To a| String |  Rename the copied file. If this value is not provided, the original file name will be kept. |  | {nbsp}
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | {nbsp}
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | {nbsp}
 |===
 
 
-==== For Configurations
+==== For Configurations.
 * <<config>> {nbsp}
 
 ==== Throws
 * SFTP:RETRY_EXHAUSTED {nbsp}
 * SFTP:ILLEGAL_PATH {nbsp}
-* SFTP:CONNECTIVITY {nbsp}
 * SFTP:FILE_ALREADY_EXISTS {nbsp}
+* SFTP:CONNECTIVITY {nbsp}
 
 
 [[createDirectory]]
 === Create Directory
 `<sftp:create-directory>`
 
-+++
-Creates a new directory on directoryPath.
-+++
+
+Creates a new directory on `directoryPath`.
```

Comment:
The Javadoc has the bad habit of using XML names instead of Studio fields. Studio users can't see the XML and XML users have no need for the Studio fields.
This line should be stated as: "Creates a new directory using the value in the **Directory Path** field."

Reply:
If this comes from Javadoc, it needs to be fixed in Javadoc. We are not spending time on this.

Reply:
Oh, I get it now. Thanks!